### DISCUSSION TITLE
Moving from builtin Redis to other

### DISCUSSION THREAD

**Discussion by fgrande** [Budibase/budibase#9360] [Help]
I'm experiencing a problem with Redis. I've changed my mind, Now I don't want to use the builtin Redis anymore, so I changed my values file adding : 

redis:
  enabled: false
  port: 6379
  replicaCount: 1
  host: "xxxxxxxxxxxxx:6379"
  password: "pppppp"

After issuing an "helm upgrade -f value.yaml....." nothing changes, and I continue on seeing the same Redis POD.

I expected to see it disappear from my namespace.... Am I wrong ? 

regards
FabioG

**Comment by shogunpurple** [comment]
hi @fgrande - it should - from the `redis-service-deployment.yaml`:

```
{{- if .Values.services.redis.enabled }}
.....
{{- end }}
```

It seems like your formatting is not quite right from the code you posted there, or is that just the way github has formatted it?

**Reply by shogunpurple** [reply]
@fgrande easier to just reply to the comment here than creating new comments.

Did you definitely deploy the update? What was the output of the log when you ran the command?

**Reply by fgrande** [reply]
Hi @shogunpurple ,

sorry for the mess with my previous answer, looks like I missed the right field to write on.

Looking at the source code, looks like I should use "url" and not "host" for giving Redis an existing path, is is right ? If this is the case, then there's a typo on https://docs.budibase.com/docs/kubernetes-k8s at Redis secion.

What would happens then if I move from builtin Redis to an existing Redis (through an helm upgrade) ? Would I loose something ? 

Thanks
FabioG

**Comment by fgrande** [comment]
Lines under "redis:" are 2 chars indented.

<img width="419" alt="image" src="https://user-images.githubusercontent.com/811941/212742822-b61d336a-d2e6-4ec8-bcb1-61282bef86f7.png">
